pkg/archive/archive_unix_test.go

```diff
@@ -248,9 +250,13 @@ func TestTarUntarWithXattr(t *testing.T) {
 		if len(changes) != 1 || changes[0].Path != "/3" {
 			t.Fatalf("Unexpected differences after tarUntar: %v", changes)
 		}
-		capability, _ := system.Lgetxattr(filepath.Join(origin, "2"), "security.capability")
-		if capability == nil && capability[0] != 0x00 {
-			t.Fatalf("Untar should have kept the 'security.capability' xattr.")
-		}
+
+		capability, err := system.Lgetxattr(filepath.Join(origin, "2"), "security.capability")
+		require.NoError(t, err)
+		assert.EqualValues(t, capability, encoded)
+
+		test, err := system.Lgetxattr(filepath.Join(origin, "1"), "user.test")
+		require.NoError(t, err)
+		assert.EqualValues(t, test, []byte("helloWord"))
```

Comment:
(For the record, the parameter order in `assert` is (expected, actual). That only matters if the test fails, and it can be trivially fixed afterwards — no need to update the PR now.)